clients/mobile/src/database/migrations/1.ts

```diff
@@ -0,0 +1,69 @@
+export default {
+  name: '1',
+  up: `
+    -- Create vault_folders table
+    CREATE TABLE IF NOT EXISTS vault_folders (
+      id TEXT PRIMARY KEY,
+      name TEXT NOT NULL,
+      "order" REAL
+    );
+
+    -- Create vaults table
+    CREATE TABLE IF NOT EXISTS vaults (
```

Comment:
can you make sure the table schema is the same as https://github.com/vultisig/vultisig-windows/blob/main/storage/migrate/000001_initial.up.sql